v0.3/cruorg/components/accordion/accordion.scss

```diff
@@ -2,3 +2,158 @@
 
 /* Accordion styles */
+
+@mixin cru-accordion {
+    &__button { 
+        @include cru-button;
+        background-color: transparent;
+        border-right: transparent;
+        border-left: transparent;
+        border-color: $cru-color-gray-light;
+        text-align: left;
+        width: 100%;
+        padding-left: 0;
+        padding-right: 0;
+        margin-bottom: -1px;
+        position: relative;
+        &:hover {
+            background-color: transparent;
+            border-color: $cru-color-gray-light;
+        }
+        &--expanded {
+            color: $cru-color-yellow;
+            font-weight: $cru-font-weight-bold;
+            .cmp-accordion__icon svg path {
+                d: path('M1 4.76367L4.764 1.08394C4.82651 1.02285 4.91128 0.988526 4.99967 0.988526C5.08806 0.988526 5.17282 1.02285 5.23533 1.08394L9 4.76367');
+                stroke: $cru-color-yellow;
+            }
+        }
+    }
+    &__panel { 
+        &--hidden { 
+            display: none;
+        }
+    }
+}
+
+@mixin cru-accordion-black {
+    &__icon {
+        svg path {
+            stroke: $cru-color-white;
+        }
+    }
+    &__button {
+        color: $cru-color-white;
+        &--expanded {
+            color: $cru-color-yellow;
+            font-weight: $cru-font-weight-bold;
+            .cmp-accordion__icon svg path {
+                stroke: $cru-color-yellow;
+            }
+        }
+    }
+    &__panel {
+        color: $cru-color-white;
+    }
+}
+
+@mixin cru-accordion-white {
+    &__button {
+        color: $cru-color-gray-dark;
+        &--expanded {
+            color: $cru-color-yellow;
+            font-weight: $cru-font-weight-bold;
+        }
+    }
+}
+
+@mixin cru-accordion-links {
+    width: 100%;
+    &__icon svg {
+        position: relative;
+        top: 10px;
+        left: 2px;
+        path {
+            stroke: $cru-color-yellow;
+        }
+    }
+    &__button {
+        background-image: linear-gradient($cru-color-yellow, $cru-color-yellow);
+        background-repeat: no-repeat;
+        background-size: 5% $cru-spacer-base / 2;
+        background-position: top left, 4px 0;
+        border: none;
+        margin: $cru-spacer-base;
+        line-height: 2rem;
+        color: $cru-color-gray-dark;
+        font-weight: $cru-font-weight-bold;
+        &--expanded {
+            .cmp-accordion__icon svg path {
+                d: path('M12 17L15.764 13.1011C15.8265 13.0364 15.9113 13 15.9997 13C16.0881 13 16.1728 13.0364 16.2353 13.1011L20 17');
+            }
+        }
+    }
+    &__panel { 
+        display: flex;
+        flex-wrap: wrap;
+        padding-left: $cru-spacer-base;
+        &--hidden {
+            display: none;
+        }
+    }
+}
+
+@mixin cru-accordion-links-black {
+    &__button {
+        color: $cru-color-white;
+    }
+}
+
+.accordion {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    padding: 5% 3%;
```

Comment:
@sebamelzi There shouldn't be any padding on this. The accordion should fill the full width of the available area.

Reply:
@sebamelzi There shouldn't be any padding here.